Units/ocaml_empty.ml_etags.t/expected.tags-e

```diff
@@ -1,2 +0,0 @@
-
-input.ml,0
```

Comment:
This is done to match behavior to other parsers, such as the C parser, which gives an empty file when fed an empty file as input.